nodejs/eks/cluster.ts

```diff
@@ -84,6 +84,8 @@ export interface CoreData {
     cluster: aws.eks.Cluster;
     vpcId: pulumi.Output<string>;
     subnetIds: pulumi.Output<string[]>;
+    publicSubnetIds: pulumi.Output<string[]>;
+    privateSubnetIds: pulumi.Output<string[]>;
```

Comment:
Seems like both `publicSubnetIds` and `privateSubnetIds` should be optional:

```typescript
    publicSubnetIds?: pulumi.Output<string[]>;
    privateSubnetIds?: pulumi.Output<string[]>;
```

Reply:
Done